weather.py

```diff
@@ -19,6 +19,12 @@ def fuzzy_direction(degrees):
     index = int(degrees) % 8
     return directions[index]
 
+def arrow_direction(degrees):
+    directions = ['\u2193', '\u2199', '\u2190', '\u2196', '\u2191', '\u2197', '\u2192', '\u2198']
```

Comment:
I think this breaks Python 2 compatibility. @Xaseron, can you make sure this works in both 2 and 3? I think this might require making these all explicit unicode literals (`u'\u2193'`) and then `.encode('utf-8')`ing the result if in Python 2.